src/main/resources/scripts/-examples/chest menus.sk

```diff
@@ -0,0 +1,55 @@
+
+#
+# An example of opening a chest inventory with an item.
+# Players will be able to take the item out.
+#
+
+command /simple chest:
+    permission: skript.example.chest
+    trigger:
+        set {_chest} to a new chest inventory named "Simple Chest"
+        set slot 0 of {_chest} to apple # Slots are numbered 0, 1, 2...
+
+#
+# An example of listening for click events in a chest inventory.
+# This can be used to create fancy button-style interfaces for users.
+#
+
+command /chest menu:
+    permission: skript.example.menu
+    trigger:
+        set {_menu} to a new chest inventory with 1 row named "Simple Menu"
+        set slot 4 of {_chest} to stone named "Button" # Slots are numbered 0, 1, 2...
```

Comment:
Wrong variable, you used `{_menu}` on the above line.
```suggestion
        set slot 4 of {_menu} to stone named "Button" # Slots are numbered 0, 1, 2...
```

Or potentially you meant both to be `{_chest}` to match the above function?

Also has the same issue as above, you should be opening `{_menu}` to the player after this.